src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  justify-content: align-items;
```

Comment:
The property 'justify-content' does not accept 'align-items' as a value. You may want to set it to 'center', 'flex-start', 'flex-end', 'space-between', 'space-around', or 'space-evenly'.